autoPyTorch/core/autonet_classes/autonet_image_data.py

```diff
@@ -113,10 +115,10 @@ def _apply_default_pipeline_settings(pipeline):
         lr_selector.add_lr_scheduler('exponential',      SchedulerExponentialLR)
         lr_selector.add_lr_scheduler('none', SchedulerNone)
         
-        lr_selector._apply_search_space_update('step:step_size', [1, 100], log=True)
-        lr_selector._apply_search_space_update('step:gamma', [0.001, 0.99], log=True)
-        lr_selector._apply_search_space_update('cosine_annealing_with_restarts:T_max', [1, 100], log=True)
-        lr_selector._apply_search_space_update('cosine_annealing_with_restarts:T_mult', [1., 2.], log=False)
+        # lr_selector._apply_search_space_update('step:step_size', [1, 100], log=True)
+        # lr_selector._apply_search_space_update('step:gamma', [0.001, 0.99], log=True)
+        # lr_selector._apply_search_space_update('cosine_annealing_with_restarts:T_max', [1, 100], log=True)
+        # lr_selector._apply_search_space_update('cosine_annealing_with_restarts:T_mult', [1., 2.], log=False)
```

Comment:
Again, should those be commented out?

Reply:
Similar reason to the above

Reply:
Similar reason to above.

Reply:
Could you please add the default values [here](https://github.com/automl/Auto-PyTorch/blob/2fb1fb77cab28127aed04d19b4a79ecf028f1c30/autoPyTorch/components/lr_scheduler/lr_schedulers.py#L289)?

Reply:
I have pushed these changes, the PR is also updated.